### ISSUE TITLE
Config Files

### ISSUE THREAD

**Issue by jasonbellamy** [plum-css/generator-plum#18]
I think we should have an option during the scaffolding process to allow the user to choose whetehr or not they want to have configuration files on the module, unit, and layout level or just use the top level config.

This will probably be helpful for smaller projects were these extra configuration files will be more of a hassle than helpful.

**Comment by jasonbellamy** [comment]
What do you think @himynameisdave?

**Comment by himynameisdave** [comment]
:-1: makes things overcomplicated a little. Like new plum users will be like "use local configs or global configs hmm fuq dis". I think by default it should just have the local configs, and if these small project-ers would rather just use the global configs then all they have to do is remove the local ones, no big deal. 

Just my opinion though, you know I like adding prompts to things, and I'm really onboard with each module/unit/page having it's own config so it's more easily used between projects === small projects get completed even faster, and perhaps will only need to change some of these config vars but ultimately re-use a lot of the same css :boom: :100:

**Comment by jasonbellamy** [comment]
I mention it because I've had a few projects come up where it would have been nice to a minimum implementation. I found myself wanting to make the tests and local configs an optional thing.

**Comment by himynameisdave** [comment]
hmm true true... 

*_forks this repo_
*_renames forked repo `generator-plum-lite`_

**Comment by himynameisdave** [comment]
You're right though, 50% of the time I'm not even gonna use the tests and I'd rather not commit them
